### DISCUSSION TITLE
Welcome to Pokedex Discussions!

### DISCUSSION THREAD

**Discussion by skydoves** [skydoves/Pokedex#20] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tuuhin** [comment]
Few months ago, I was analyzing this project, I found out that ,
`https://assets.pokemon.com/assets/cms2/img/pokedex/full/{pokemon_id}.png` for the sprites.
Today for some reason I was hovering over the pokeapi , found out that ,It now provides better sprites 
- The old sprite `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/{pokemon_id}.png` 
That wasn't a bit cool 

The others that I found today 
- SVG 🖼️ 
 `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/dream-world/{pokemon_id}.svg`
- Stuff Toy like 🧥 
`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/home/{pokemon_id}.png`
- Official Artwork as in [Bulbapedia](https://bulbapedia.bulbagarden.net/wiki/Bulbasaur_(Pok%C3%A9mon))  🎨 
- `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/{pokemon_id}.png`

For more information analyze **sprite** for [pokemon](https://pokeapi.co/api/v2/pokemon/{poke_id}/)

Hope it was helpful...

**Comment by isaacggr** [comment]
Hey, thanks for sharing these discoveries!

You're right, these new sprites offer much better quality and are great options for anyone looking for a more detailed look for the Pokédex. I've also noticed the improvement in the variety of formats, like the SVG, which allows for scalability without losing quality.

The official artwork is another highlight, giving it a more authentic look, like what we see on Bulbapedia. These new options could really enhance the user experience, offering a variety of images for different style preferences.

I'm going to try implementing these alternatives in my own Pokédex project, and I believe others will like the idea too. Thanks for sharing!

**Reply by SDBoii** [reply]
Check out this Pokédex project as well...It's pretty creative and might help spark up new ideas🙂

https://github.com/Sumdiboii/PokeDex-Project

Do let me know if you want the source code💯
